persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/Conversion.java

```diff
@@ -162,6 +164,20 @@
     TYPE_MAPPINGS = ImmutableMap.copyOf(types);
   }
 
+  private static ByteBuffer getKeyValue(
```

Comment:
is there no chance that all these duplicates are extracted to some to a common persistence.. Is there no chance that at least some of these depending classes are shared between all persistence modules?

Reply:
Yes, I was thinking about moving it to `PagingPosition`, but the trick is that all the related table and column classes are slightly different in each persistence implementation and cannot be shared. I suppose we could use `String` arguments instead. I'll try and see what it looks like.

Reply:
Moved common code to `PagingPosition`... Does it look better this way? :)